openvdb_points/tools/AttributeArray.h

```diff
@@ -1506,70 +1582,88 @@ AttributeHandle<T>::AttributeHandle(const AttributeArray& array, const bool pres
     mFiller = typedAccessor->mFiller;
 }
 
+template <typename T, bool Strided, bool Interleaved>
+Index AttributeHandle<T, Strided, Interleaved>::index(Index n, Index m) const
+{
+    if (Strided) {
+        if (Interleaved)    return m * size() + n;
+        else                return n * stride() + m;
```

Comment:
These are mainly template parameters to ensure that on get or set for non-strided access, it is not necessary to check a bool at runtime on each access.